tools/ansible/tasks/logging/install_plugins.yml

```diff
@@ -0,0 +1,20 @@
+---
+# Installing plugins listed here: https://github.com/openshift/origin-aggregated-logging/blob/master/fluentd/Gemfile 
```

Comment:
I am fine with this for now but going forward, we need to somehow get the list from the repo or the plugins installed on the linux host and get the plugin names from there.

Reply:
I agree, we should address this in a future commit